src/Setup/BuildPlan.purs

```diff
@@ -52,19 +49,24 @@ getVersionField key = do
         pure (pure (Exact version))
 
 -- | Resolve the exact version to provide for a tool in the environment, based
--- | on the action.yml file.
-resolve :: Json -> Tool -> ExceptT Error Effect (Maybe { tool :: Tool, version :: Version })
+-- | on the action.yml file. In the case of NPM packages, bypass the action.yml
+-- | file and use 'Latest'.
+resolve :: Json -> Tool -> ExceptT Error Effect (Maybe { tool :: Tool, versionField :: VersionField })
 resolve versionsContents tool = do
   let key = Key.fromTool tool
   field <- getVersionField key
```

Comment:
😕 Don't feel good about this approach where I get a `VersionField` within this function, but then do some transformation of it before returning a `VersionField` as output.